en-US/firefox/welcome/page7.lang

```diff
@@ -0,0 +1,75 @@
+## NOTE: demo page available at https://www-dev.allizom.org/firefox/welcome/7/
```

Comment:
The link doesn't work

Reply:
The link will work in just a bit :D we haven't push to dev yet but it will be there today.

Reply:
The langchecker PR is stuck with Travis. I can't merge it.

Reply:
The URL still doesn't work this morning.